src/Stratis.SmartContracts.Token/StandardToken.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Stratis.SmartContracts;
+using Stratis.SmartContracts.Token;
+
+public class StandardToken : SmartContract, IStandardToken
```

Comment:
I think this class should just have a brief summary stating how it works - primarily what TotalSupply means and who receives the tokens on deployment.